front/style/layout/_buttons.scss

```diff
@@ -191,6 +191,17 @@
     }
 }
 
+@mixin btn--ghosting--blue {
```

Comment:
What does `ghosting` stand for? Is it a naming that people will understand?

Reply:
this is for buttons with transparent background. today only default one (black) are existing and they use the default clzass `.btn`, i've added the blue one.